skiplang/prelude/ts/src/sk_worker.ts

```diff
@@ -241,7 +241,7 @@ export class PromiseWorker {
     this.onMessage = (message: MessageEvent) => {
       let data = Message.asReturn(message.data ?? message);
       if (!data) {
-        throw new UnmanagedMessage(message);
+        throw new UnmanagedMessage(JSON.stringify(message));
```

Comment:
I'm not sure this is the best way to create a string from a MessageEvent. Any suggestions?